corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -97,7 +97,10 @@ def rate_limit_two_factor_setup(device):
     then those requests are also rejected.
 
     """
-    _status_rate_limited = 'rate_limited'
+    _status_global_rate_limited = 'global_rate_limited'
+    _status_ip_rate_limited = 'ip_rate_limited'
+    _status_number_rate_limited = 'number_rate_limited'
+    _status_user_rate_limited = 'user_rate_limited'
     _status_bad_request = 'bad_request'
```

Comment:
Rather than create a new metric for 'reason', it seemed to make more sense to expand upon the status metric.

Reply:
Cool, I like that